src/ResourceManager/Websites/Commands.Websites.Test/ScenarioTests/WebAppSlotTests.ps1

```diff
@@ -43,7 +43,7 @@ function Test-GetWebAppSlot
 		Assert-AreEqual $serverFarm.Id $webapp.ServerFarmId
 		
 		# Create new deployment slot
-		$slot1 = New-AzureRmWebAppSlot -ResourceGroupName $rgname -Name $appname -Slot $slotname1 -AppServicePlan $planName 
+		$slot1 = New-AzureRmWebAppSlot -ResourceGroupName $rgname -Name $appname -Slot $slotname1 -AppServicePlan $planName
```

Comment:
Looks like you didn't add the test here.

Reply:
This is the Test-GetWebAppSlot , i haven't updated this one i add it to the Test-CreateNewWebAppSlot

Reply:
So in Test-CreateNewWebAppSlot, I see you updated the New-AzureRmWebApp cmdlet but not the New-AzureRmWebAppSlot cmdlet, so at the moment there is no test for New-AzureRmWebAppSlot.

Reply:
Thanks for pointing this out - updated now